object-manipulation-2/src/invert.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+function invert(source) {
+  var reversedObject = {};
+  var keys = [];
+  var values = [];
+
+  for (var key in source) {
+    keys.push(key);
+    values.push(source[key]);
+  }
```

Comment:
You can modify an object using the object data itself, rather than splitting the object into two arrays.